server.js

```diff
@@ -710,10 +710,24 @@ module.exports.initExpress = function() {
         require('./pages/studentAssessmentInstanceHomework/studentAssessmentInstanceHomework'),
         require('./pages/studentAssessmentInstanceExam/studentAssessmentInstanceExam'),
     ]);
+    app.use('/pl/course_instance/:course_instance_id/instructor/assessment/:assessment_id/assessment_question/:assessment_question_id/unlock_ungraded', [
```

Comment:
This is an instructor route but it's in the section for student routes: https://github.com/PrairieLearn/PrairieLearn/blob/5483eb8c5698e3b3e5b7e888c1d9b4e51434e270/server.js#L672

See also this earlier comment: https://github.com/PrairieLearn/PrairieLearn/pull/3707/files#r565474285

Reply:
Done!